src/RoadRegistry.BackOffice.ZipArchiveWriters/ExtractHost/IntegrationToZipArchiveWriter.cs

```diff
@@ -196,6 +183,7 @@ async Task WriteRoadSegments()
             }
 
             await archive.CreateCpgEntry("iWegsegment.cpg", _encoding, cancellationToken);
```

Comment:
Shouldn't we best also replace the filename strings here using the methods like at `FeatureType.Integration.ToProjectionFileName(ExtractFileName.Wegsegment)` so it's all the same?